ddtrace/contrib/yaaredis/patch.py

```diff
@@ -0,0 +1,66 @@
+import yaaredis
+
+from ddtrace import config
+from ddtrace.vendor import wrapt
+
+from ...pin import Pin
+from ...utils.wrappers import unwrap
```

Comment:
Have to update this once #2938 is merged

Reply:
```suggestion
from ...internal.utils.wrappers import unwrap
```

Reply:
It's not merged, you confused me with your suggest 😢

Reply:
something to keep an eye on, I was expecting #2938 to get merged before this, don't want to have this merged and then have it log a deprecation warning

Reply:
ok, now this needs to be changed. #2938 has been merged.